nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -20,6 +20,16 @@ In addition to numerous new and upgraded packages, this release has the followin
   built for `stdenv.hostPlatform` (i.e. produced by `stdenv.cc`) by evaluating
   `stdenv.buildPlatform.canExecute stdenv.hostPlatform`.
 
+- The `polymc` package has been removed due to a rogue maintainer. It has been
```

Comment:
May I ask why this is under the "highlights" section of our release notes? Yes, I fully agree that the incident is fairly unfortunate, but at the end of the day this is just a small leaf-package affecting a minority of NixOS installations, so it should be under "breaking changes" or "other notable changes" IMHO.

Reply:
You're right, it probably should be. I will make another PR to move it

Reply:
Will be fixed in PR #197500